seatunnel-config/src/test/resources/equiv01/no-whitespace.json

```diff
@@ -1 +1,49 @@
-{"ints":{"fortyTwo":42,"fortyTwoAgain":42},"floats":{"fortyTwoPointOne":42.1,"fortyTwoPointOneAgain":42.1},"strings":{"abcd":"abcd","abcdAgain":"abcd","a":"a","b":"b","c":"c","d":"d","concatenated":"null bar 42 baz true 3.14 hi"},"arrays":{"empty":[],"1":[1],"12":[1,2],"123":[1,2,3],"ofString":["a","b","c"]},"booleans":{"true":true,"trueAgain":true,"false":false,"falseAgain":false},"nulls":{"null":null,"nullAgain":null}}
\ No newline at end of file
+{
+  "ints": {
+    "fortyTwo": 42,
+    "fortyTwoAgain": 42
+  },
+  "floats": {
+    "fortyTwoPointOne": 42.1,
+    "fortyTwoPointOneAgain": 42.1
+  },
+  "strings": {
+    "abcd": "abcd",
+    "abcdAgain": "abcd",
+    "a": "a",
+    "b": "b",
+    "c": "c",
+    "d": "d",
+    "concatenated": "null bar 42 baz true 3.14 hi"
+  },
+  "arrays": {
+    "empty": [],
+    "1": [
+      1
+    ],
+    "12": [
+      1,
+      2
+    ],
+    "123": [
+      1,
+      2,
+      3
+    ],
+    "ofString": [
+      "a",
```

Comment:
just a nit: the following form is friendly.
```
"123": [1, 2, 3]
```

Reply:
fixed